src/ui/screens/Parachains/CrowdloanScreen.tsx

```diff
@@ -190,10 +191,12 @@ function ContributeBox({
   visible,
   setVisible,
   parachainId,
+  refetch,
 }: {
   visible: boolean;
   setVisible: (_visible: boolean) => void;
   parachainId: string;
+  refetch: () => void;
```

Comment:
This name is a bit confusing in the context of this component, you can call it `onContribute`. It does not need to know that will refetch some query